<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DVSstartUpPage</class>
 <widget class="QMainWindow" name="DVSstartUpPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>711</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DVS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>280</y>
      <width>391</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="MiddlegridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="CreateNew_pushButton">
       <property name="text">
        <string>Create New Project</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="CurrentProject_PushButton">
       <property name="text">
        <string>Open Previous Project</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="Settings_pushButton">
       <property name="text">
        <string>Settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>1011</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Top_verticalLayout">
     <item>
      <spacer name="Top_verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>1011</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Bottom_verticalLayout">
     <item>
      <spacer name="Bottom_verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>279</y>
      <width>301</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="Left_horizontalLayout">
     <item>
      <spacer name="Left_horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>280</y>
      <width>301</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="Right_horizontalLayout">
     <item>
      <spacer name="Right_horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1025</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
